meinberlin/apps/bplan/templates/meinberlin_bplan/bplan_list_dashboard.html

```diff
@@ -8,7 +8,7 @@
         <h1 class="lr-bar__left">
             {% trans 'Development plans' %}
         </h1>
-        <div class="lr-bar__right">
+        <div class="lr-bar__mobile-left-desk-right">
```

Comment:
is that class name too much? I have bad Monday brain, might re-asses in morning

Reply:
Haha, I needed a while. You could write out desktop?! Oh no: sm-left-lg-right maybe?